src/patterns/FavoritesCollectionTile/FavoritesCollectionTile.stories.tsx

```diff
@@ -18,10 +18,11 @@ Playground.args = {
   count: 2,
   name: 'New List',
   imageSrc: 'https://via.placeholder.com/400',
-  variant: 'favorites',
+  variant: 'lists',
   onEdit: (): void => console.log('Edit list clicked'),
   onDelete: (): void => console.log('Delete list clicked'),
   element: 'a',
+  linkClassName: 'seldon-link seldon-link--link',
```

Comment:
We should still be using the `px` variable for your class names in case that changes ever. Unlikely but still the practice.